setuptools_scm find_files tries to create archive from whole oi-userland...
--- urllib3-1.25.3/setup.py     2019-05-23 22:01:31.000000000 +0000
+++ urllib3-1.25.3/setup.py     2019-07-30 09:32:48.804949843 +0000
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from setuptools_scm import integration
+integration.find_files = lambda p: []
 
 import os
 import re
